src/main/java/org/openRealmOfStars/player/PlayerInfo.java

```diff
@@ -453,9 +453,11 @@ public PathPoint getUnchartedSector(final Sun sun, final Fleet fleet) {
           if (mapData[sun.getCenterX() + x][sun.getCenterY()
               + y] == UNCHARTED) {
             unCharted[sector]++;
+            Coordinate fleetCoordinate = new Coordinate(fleet.getX(), fleet.getY());
+            //@TODO: This code snippet is confusing. I am not sure why is this coordinate increased by x and y.
```

Comment:
This method is used to calculate sector coordinate where would be most of uncharted tiles between fleet and sector coordinate in solar system which is being explored. Sun coordinate is increased by X and Y to get that sector coordinate. Since sun's coordinates are absolute coordinates and X and Y are being cycled through in for loops.